guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>The training results are printed using <code>model.show_results()</code> to assess the quality of the trained model and also compute model metrics. The <code>model.show_results()</code> will <u>typically</u> output a table where <s>usually </s>the last column will <s>return</s> <u>contain </u>the values predicted by the model after being applied to the test dataset. <s>predicted values by the model on the test set</s>. Comparing the predicted values with the actual target variable will give some indication of the performance of the trained model. Subsequently, the model metrics of the trained model <s>could</s> <u>can </u>be estimated using the <code>model.score()</code> function which currently returns the r-square, which<s> will</s> also reflects model performance.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>